gpu_stats/src/main.rs

```diff
@@ -325,26 +324,23 @@ impl SystemMonitorService for SystemMonitorServiceImpl {
         &self,
         request: Request<GetMetadataRequest>,
     ) -> Result<Response<GetMetadataResponse>, Status> {
-        debug!("Received a request to get metadata: {:?}", request);
+        debug!("Received a GetMetadata request: {:?}", request);
```

Comment:
should we remove this one? i thought you removed this request

Reply:
it's a grpc request between core and gpu_stats